source/features/improve-shortcut-help.tsx

```diff
@@ -45,17 +46,19 @@ const observer = new MutationObserver(([{target}]) => {
 	}
 });
 
-function init(): void {
-	document.addEventListener('keypress', ({key, target}) => {
-		if (key !== '?' || target instanceof HTMLTextAreaElement || target instanceof HTMLInputElement) {
-			return;
-		}
+function observeShortcutModal({key, target}: KeyboardEvent): void {
+	if (key !== '?' || isEditable(target)) {
+		return;
+	}
 
-		const modal = select('body > details > details-dialog');
-		if (modal) {
-			observer.observe(modal, {childList: true});
-		}
-	});
+	const modal = select('body > details > details-dialog');
+	if (modal) {
+		observer.observe(modal, {childList: true});
+	}
+}
+
+function init(): void {
+	document.addEventListener('keypress', observeShortcutModal);
```

Comment:
Does this work if you press `?` after an ajax navigation?
```suggestion
	document.addEventListener('keypress', observeShortcutModal, {once: true});
```

Reply:
No